wolfSSL/src/chapter02.md

```diff
@@ -394,6 +394,122 @@ Removes code for OAEP padding.
 
 Turns off AES-CBC algorithm support.
 
+#### WOLFCRYPT_ONLY
+
+Enables wolfCrypt only while disabling TLS.
+
+#### NO_CAMELLIA_CBC
+
+Disables Camellia CBC support.
+
+#### NO_AES
+
+Disables AES algorithm support.
+
+#### NO_AESGCM_AEAD
+
+Disables AES GCM algorithm support.
+
+#### NO_ASN_TIME
+
+Disables time checking for ASN.
```

Comment:
This should be used with caution because all certificate begin/end date checking will be skipped.